gradle.properties

```diff
@@ -2,10 +2,10 @@
 org.gradle.jvmargs=-Xmx1G
 # Fabric Properties
 # check these on https://fabricmc.net/use
-minecraft_version=1.16.3
-yarn_mappings=1.16.3+build.11
-loader_version=0.9.3+build.207
+minecraft_version=1.16.5
+yarn_mappings=1.16.5+build.5
+loader_version=0.11.2
```

Comment:
its not neccessary,because this mod will support many versions, not only for the latest version.

Reply:
Can be reverted then.